blogpost_14.html

```diff
@@ -324,6 +324,13 @@ <h2><a href="https://docs.xfce.org/xfce/xfwm4/start" target="_blank">xfwm4</a></
             <p>The theme "Greybird-dark" and the icon-theme "elementary Xfce dark" were used for the screenshots. This summary can as well be viewed as a <a href="https://www.xfce.org/about/tour420">localized online tour</a>.</p>
             <p>This release <a href="https://archive.xfce.org/xfce/4.20">can be downloaded here</a>, either as a set of individual packages or as a single tarball including all individual versions.</p>
 
+            <h2>Our Sponsors</h2>
+            <div class="t1">
+              <p>Thanks go to <a href="https://www.gandi.net" target="_blank">Gandi</a> for providing the Xfce server infrastructure for free, and to <a href="https://kalmarndc.se" target="_blank">Kalmar NDC</a> for sponsoring our GitLab runners. We appreciate your support!</p>
+              <p>As well many thanks to our generous financial supporters on <a href="https://opencollective.com/xfce" target="_blank">Open Collective US</a> and <a href="https://opencollective.com/xfce-eu" target="_blank">Open Collective EU</a>!</p>
+              <p>As a team of volunteers we have no concrete idea yet on how to use the fonds, however it`s great to know that we have strong financial backing. Thank you for your support!</p>
```

Comment:
Suggestion: At the moment, aside from the couple of hardware purchases for our developers, we don’t have specific plans for the funds. We are exploring the best ways to use them effectively. It's reassuring to know we have such strong financial support. Thank you for your support!

Reply:
Thanks! Tweaked accordingly  (I just replaced the double "support" at the end)